src/edit.js

```diff
@@ -42,3 +50,10 @@ export default function Edit( { attributes, setAttributes } ) {
 		</div>
 	);
 }
+
+const Edit = withColors( 'color', {
+	messageColor: 'color',
+	progressColor: 'color',
+} )( ProgressEdit );
+
```

Comment:
If you switched to a pure Gutenberg implementation, this shouldn't be needed anymore.

However, I don't think we fully switched yet, as we still need both anyway, but this isn't consumed above, so it's just sitting here adding attributes for nothing.

Reply:
I decided to switch to a pure Gutenberg implementation as it works in all themes, while the previous implementation did not work with all themes, such as Storefront.